### PR TITLE
Clean nightscout

### PR DESCRIPTION
**Pull request by tzachi-dar**
After working with this project for a few month, my nightscout site has stopped working since my mongodb was left without space.

Since this probably happens to everyone, I have created a script that goes directly to the mongodb and deletes objects older than two months.

This script probably needs to be moved somewhere else, and we need to find a way to get the db, user and password (maybe we will have to write it to nightscout instead?).

Looking forward for all comments.

### CODE REVIEW

File: bin/CleanNightscout.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import json
+import time
+
+import datetime
+from bson import json_util
+from pytz import timezone
+
+
+db_uri = 'mongodb://user:pass@ds135003.mlab.com:35003'
+db_name = 'heroku_b6k1n0tj'
```
[the remark this answers is not in the document]
Comment (tzachi-dar) [reply bin/CleanNightscout.py]: Well, currently, the user will have to replace this strings, which is not ideal.
If there are no other suggestions, than we need to add instructions on how to take this parameters from the settings in the mongo account.
Ideally, we should get access based on the apisec that we already store on the rig.

I guess we can start with explanation. Let me know if that sounds reasonable, and I'll add the explanation.

**Comment by jpcunningh** [comment]
The `dev` branch of Nightscout has an API you can use to delete data from entries, treatments, and devicstatus rather than having to go directly to Mongo. The `dev` branch should be released to `master` within the next couple of weeks.

I would recommend using the NS API instead of the Mongo direct connection.

**Comment by tzachi-dar** [comment]
I guess that using the new api will make things simpler for the users, so let's wait for the new api.

**Comment by scottleibrand** [comment]
Nightscout just got released: should we update this PR, or close it and do a new one?

**Comment by tzachi-dar** [comment]
I'm closing it now, will do a new one against the better api.

**Comment by sulkaharo** [comment]
Note you can also get "near infinite" amount of space for $5 / month by installing Mongo to Digital Ocean.

**Comment by tzachi-dar** [comment]
Indeed tumpteing.
Probably will save me some time, but this makes installation for everyone more complicated.
Thanks for the advice @sulkaharo

**Pull request closed without merging**